pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/TopicStats.java

```diff
@@ -281,6 +281,16 @@ static void printTopicStats(SimpleTextOutputStream stream, String cluster, Strin
                     subsStats.msgDropRate, splitTopicAndPartitionIndexLabel);
             metric(stream, cluster, namespace, topic, n, "pulsar_subscription_consumers_count",
                     subsStats.consumersCount, splitTopicAndPartitionIndexLabel);
+
+            metric(stream, cluster, namespace, topic, n, "pulsar_subscription_through_filter_msg_count",
```

Comment:
Same comment above, but here it should to be consistent `pulsar_subscription_filter_processed_msgs_count`